package.json

```diff
@@ -5,7 +5,8 @@
   "dependencies": {
     "react": "^16.8.2",
     "react-dom": "^16.8.2",
-    "react-redux": "^6.0.0"
+    "react-redux": "^6.0.0",
+    "webdriverio": "^4.0.7"
```

Comment:
`webdriverio` should be in `devDependencies`

Reply:
Fixed